.github/workflows/stale.yml

```diff
@@ -0,0 +1,31 @@
+name: Stale PRs
+on:
+  schedule:
+    - cron: '0 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - name: Label/close stale PRs
+        uses: actions/stale@v8
+        with:
+          stale-pr-message: 'This PR is stale because it has been open 60 days with no activity. Remove stale label or comment or this will be closed in 10 days.'
+          close-pr-message: 'This PR was closed because it has been stalled for 10 days with no activity.'
+
+          # Stale after 60 days, close after further 10 days
+          days-before-pr-stale: 60
+          days-before-pr-close: 10
+
+          # Don't work on issues (only PRs)
+          days-before-issue-stale: -1
+          days-before-issue-close: -1
+
+          # Only consider 'community' PRs
+          only-labels: 'community'
+
+          # Delete its branch after closing a PR
+          delete-branch: true
```

Comment:
This might not be possible in case of a fork, I guess, but community PRs would probably mostly be created from forks? In any case I am not completely sure if we wanted to delete any branches automatically, that could be painful in case a PR is closed accidentally if for example a person is on a longer leave or similar.

Reply:
Otherwise, we'll have to deal with all the zombie branches manually and that's a bit frustrating because we'll need to ping people, read through a lot of commit history etc. to be able to safely delete them.

AFAIK, GH backs up deleted branch heads so it should be possible to restore them.

Forked branches won't be deleted with this action, but since they're not in our domain, we shouldn't care about them. This setting is only meant as a cleanup for branches created in our repo, so it's only relevant for PRs from us.